db/views/patient_summaries_v01.sql

```diff
@@ -0,0 +1,88 @@
+SELECT
+p.recorded_at,
+CONCAT(date_part('year', p.recorded_at), ' Q', EXTRACT(QUARTER FROM p.recorded_at)) AS registration_quarter,
+p.full_name,
+(
+    CASE
+      WHEN p.date_of_birth IS NOT NULL THEN date_part('year', age(p.date_of_birth))
+      ELSE p.age + date_part('year', NOW()) - date_part('year', p.age_updated_at)
+    END
+) AS current_age,
+p.gender,
+latest_phone_number.number AS latest_phone_number,
+addresses.village_or_colony,
+addresses.district,
+addresses.state,
+reg_facility.name AS registration_facility_name,
+reg_facility.facility_type AS registration_facility_type,
+reg_facility.district AS registration_district,
+reg_facility.state AS registration_state,
+latest_bp.systolic AS latest_bp_systolic,
+latest_bp.diastolic AS latest_bp_diastolic,
+latest_bp.recorded_at AS latest_bp_recorded_at,
+CONCAT(date_part('year', latest_bp.recorded_at), ' Q', EXTRACT(QUARTER FROM latest_bp.recorded_at)) AS latest_bp_quarter,
+latest_bp_facility.name AS latest_bp_facility_name,
+latest_bp_facility.facility_type AS latest_bp_facility_type,
+latest_bp_facility.district AS latest_bp_district,
+latest_bp_facility.state AS latest_bp_state,
+greatest(0, date_part('day', NOW() - next_appointment.scheduled_date)) AS days_overdue,
+next_appointment.scheduled_date AS next_appointment_scheduled_date,
+next_appointment_facility.name AS next_appointment_facility_name,
+next_appointment_facility.facility_type AS next_appointment_facility_type,
+next_appointment_facility.district AS next_appointment_district,
+next_appointment_facility.state AS next_appointment_state,
+
+(
+    CASE
```

Comment:
The risk level calculations don't take into account one of the scenarios mentioned [here](https://docs.simple.org/engineering/backend/major-flows#high-risk-label-in-overdue-list): when the patient's latest scheduled appointment is overdue under a month. Currently implemented in [this](https://github.com/simpledotorg/simple-server/blob/master/app/models/patient.rb#L99).

I believe another scenario where it's overdue > 365 days is moot since overdue itself now is capped to 365 days. Currently implemented [here](https://github.com/simpledotorg/simple-server/blob/master/app/models/patient.rb#L180).